components/Notification.js

```diff
@@ -0,0 +1,33 @@
+import { Toaster } from "react-hot-toast";
+import styled from "styled-components";
+
+export default function Notification() {
+  return (
+    <Toaster
+      position="top-center"
+      reverseOrder={false}
+      gutter={8}
+      containerClassName=""
+      containerStyle={{}}
+      toastOptions={{
+        // Define default options
+        className: "",
+        duration: 3500,
+        style: {
+          background: "#363636",
+          color: "#fff",
```

Comment:
I don't know the library, but have you tried styling the toaster via styled component or does that not work?

Reply:
Hi Feline, 

in this case we relied on the default usage of the library - just like the other group. Theoretically it would be possible, but in this case we need to write the whole logic of the library from scratch with the custom hook „useToast“. This is also possible and we tried at first, but as this seems to be a lot more work, we added this on our backlog list for next week.